src/components/Button/Button.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+
+interface ButtonProps extends React.ButtonHTMLAttributes<HTMLButtonElement> {
+  children: React.ReactNode;
+  name: string;
+}
+
+const Button: React.FC<ButtonProps> = ({ children, name, ...props }) => {
+  return (
+    <>
+      <button
+        type="button"
+        name={name}
+        {...props}
```

Comment:
This should be placed as the last item after all the other attributes to allow the consumer of this component to override any of the values.